### PR TITLE
Feat/delete destination

### PR DESCRIPTION
**Pull request by sebsch1337**
You can now delete destinations. Please check it out ;)

### CODE REVIEW

File: pages/destinations/[id].js [hunk 1]
```diff
@@ -68,38 +73,51 @@ export default function Destinations() {
           {destinations
             .filter((destination) => destination.tripId === id)
             .map((item) => (
-              <DestinationItem key={item.id} id={item.id} name={item.name} />
+              <DestinationItem
```
Comment (ThomasF85) [inline pages/destinations/[id].js:76]: The prop onDeleteDestination is not being handed over to this component

Comment (ThomasF85) [reply pages/destinations/[id].js:76]: ..ok, no need, the click opens a modal which might then delete the destination

File: components/DestinationItem.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
 import bulletCircle from "../assets/bulletCircle.svg";
 import styled from "styled-components";
 import Link from "next/link";
+import DeleteButton from "./DeleteButton";
 
-export default function DestinationItem({ id, name }) {
+export default function DestinationItem({ id, name, onClick, onDeleteDestination }) {
```
Comment (ThomasF85) [inline components/DestinationItem.js:6]: unused prop onDeleteDestination

**Review by ThomasF85 — APPROVED** [review]
Nice! Just remove the unused prop in DestinationItem please!

File: components/DestinationItem.js (hunk 2 above)
Comment (sebsch1337) [reply components/DestinationItem.js:6]: Removed it. Thanks!

**Pull request merged**